src/test/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrFilmDetailTaskTest.java

```diff
@@ -0,0 +1,256 @@
+                  "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/148/1480611/,1480611_16974214,1480611_16974213,1480611_16974215,1480611_16974211,1480611_16974212,.mp4.csmil/index_0_av.m3u8",
+                  "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/148/1480611/,1480611_16974214,1480611_16974213,1480611_16974215,1480611_16974211,1480611_16974212,.mp4.csmil/index_2_av.m3u8",
+                  "",
+                  new GeoLocations[] {GeoLocations.GEO_NONE})
+            }
+          },
+          {
+            new TestData[] {
+              new TestData(
+                  "http://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet---schokolade-102.html",
+                  "/wdr/wdr_film2.html",
+                  "/ondemand/140/1407842.js",
+                  "/wdr/wdr_video2.js",
+                  "/i/medp/ondemand/weltweit/fsk0/140/1407842/,1407842_16309723,1407842_16309728,1407842_16309725,1407842_16309726,1407842_16309724,1407842_16309727,.mp4.csmil/master.m3u8",
+                  "/wdr/wdr_video2.m3u8",
+                  "Ausgerechnet",
+                  "Ausgerechnet - Schokolade",
+                  LocalDateTime.of(2017, 7, 15, 16, 0, 0),
+                  Duration.ofMinutes(43).plusSeconds(35),
+                  "Knapp 25 Prozent der Bevölkerung verzehren Schokolade mehrmals in der Woche. Hinzu kommt gut ein weiteres Viertel, das Schokolade etwa einmal pro Woche genießt. Frauen greifen hier generell häufiger zu als Männer.",
+                  "http://wdrmedien-a.akamaihd.net/medp/ondemand/weltweit/fsk0/140/1407842/1407842_16348809.xml",
+                  "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/de/fsk0/140/1407842/,1407842_16309723,1407842_16309728,1407842_16309725,1407842_16309726,1407842_16309724,1407842_16309727,.mp4.csmil/index_0_av.m3u8",
+                  "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/de/fsk0/140/1407842/,1407842_16309723,1407842_16309728,1407842_16309725,1407842_16309726,1407842_16309724,1407842_16309727,.mp4.csmil/index_2_av.m3u8",
+                  "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/de/fsk0/140/1407842/,1407842_16309723,1407842_16309728,1407842_16309725,1407842_16309726,1407842_16309724,1407842_16309727,.mp4.csmil/index_4_av.m3u8",
+                  new GeoLocations[] {GeoLocations.GEO_DE})
+            }
+          },
+          {
+            new TestData[] {
+              new TestData(
+                  "http://www1.wdr.de/mediathek/video/sendungen/wdr-aktuell/video-wdr-aktuell-988.html",
+                  "/wdr/wdr_film_with_part.html",
+                  "/ondemand/158/1583472.js",
+                  "/wdr/wdr_film_with_part.js",
+                  "/i/medp/ondemand/weltweit/fsk0/158/1583472/,1583472_18232890,1583472_18232889,1583472_18232891,1583472_18232887,1583472_18232888,.mp4.csmil/master.m3u8",
+                  "/wdr/wdr_film_with_part.m3u8",
+                  "WDR aktuell",
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "WDR aktuell" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)